bpf/lib/lb.h

```diff
@@ -266,7 +267,11 @@ bool lb6_svc_has_src_range_check(const struct lb6_service *svc __maybe_unused)
 
 static __always_inline bool lb_skip_l4_dnat(void)
 {
+#ifdef ENABLE_DSR_TUNL
```

Comment:
Again, I wonder, do we need a configurable option for this?
Could we use `defined(TUNNEL_MODE) && (DSR_ENCAP_MODE == DSR_ENCAP_IPIP)`?